R/namespace.R

```diff
@@ -75,5 +92,16 @@ is_s3_generic <- function(fun) {
     # R < 3.5.0 doesn't provide .S3_methods_table
     # fallback: search baseenv() for generic methods
     imported_s3_generics(data.frame(pkg = "base", fun = ls(baseenv()), stringsAsFactors = FALSE))$fun
-  }
-)
+  },
+  # S3 generic groups, see ?base::groupGeneric
+  # Group "Math"
+  c("abs", "sign", "sqrt", "floor", "ceiling", "trunc", "round", "signif", "exp", "log", "expm1", "log1p", "cos",
```

Comment:
Missing `log2()` and `log10()` right?

https://github.com/r-devel/r-svn/blob/bb4492e2a913a187ae77fe1f81329a2c81c320a7/src/library/base/R/zzz.R#L178-L184

Reply:
WDYT about using `ls(.GenericArgsEnv)` instead? That would also subsume `.S3PrimitiveGenerics`.

Reply:
Sounds good. I'll try this and see if all R versions work with that.

Reply:
Update: Looks like it works. Thanks for the tip!